app/car.py

```diff
@@ -0,0 +1,49 @@
+import json
+import os
+import math
+from app.shop import Shop
+
+
+current_directory = os.path.dirname(os.path.abspath(__file__))
+relative_path = os.path.join(current_directory, "config.json")
+customers = []
+with open(relative_path, "r") as file:
+    content = json.load(file)
+
+
+class PriseKm:
+    def __init__(self, content: dict) -> list:
+        self.content = content
+
+    def distens_prise() -> dict:
```

Comment:
Word distans and prise does not exist
https://context.reverso.net/translation/english-ukrainian/price
https://context.reverso.net/translation/english-ukrainian/distance